modules/calcite/src/test/java/org/apache/ignite/internal/processors/tx/SqlTransactionsIsolationTest.java

```diff
@@ -101,7 +103,10 @@ public enum ExecutorType {
         CLIENT,
 
         /** */
-        THIN
+        THIN_VIA_CACHE_API,
```

Comment:
Why thin client is created via private API TcpIgniteClient and not by Ignition.startClient?

Reply:
Fixed.